packages/sdk-client-v3/src/middleware/create-logger-middleware.ts

```diff
@@ -18,18 +18,17 @@ export default function createLoggerMiddleware(
 
       const {
         loggerFn = console.log,
-        // logLevel = 'ERROR',
         maskSensitiveHeaderData = true,
         includeOriginalRequest = true,
         includeResponseHeaders = true,
-        // includeRequestInErrorResponse
       } = options || {}
 
-      if (includeOriginalRequest && maskSensitiveHeaderData) {
-        maskAuthData(response.request)
-      }
-
-      if (!includeOriginalRequest) {
+      if (includeOriginalRequest) {
+        response.request = request
```

Comment:
Why did you have to introduce this line?

Reply:
Because `response.request` was undefined when I did the integration test for this.